lib/Backend/GlobOpt.cpp

```diff
@@ -5488,6 +5489,21 @@ GlobOpt::IsPrepassSrcValueInfoPrecise(IR::Opnd *const src, Value *const srcValue
         !currentBlock->loop->regAlloc.liveOnBackEdgeSyms->Test(srcSym->m_id);
 }
 
+bool
+GlobOpt::IsSafeToTransferInPrepass(StackSym * const srcSym, ValueInfo *const srcValueInfo) const
+{
+    int32 intConstantValue;
+    return
+        srcSym->IsFromByteCodeConstantTable() ||
+        (
+            srcValueInfo->TryGetIntConstantValue(&intConstantValue) &&
+            !Js::TaggedInt::IsOverflow(intConstantValue) &&
+            GetTaggedIntConstantStackSym(intConstantValue) == srcSym
```

Comment:
I see the same code above...  What is it about?

Reply:
I will be reusing this code in the above function in a subsequent PR. Just wanted to change the object-pointer-copy-prop semantics in this PR.

---
In reply to: [180214048](https://github.com/Microsoft/ChakraCore/pull/4943#discussion_r180214048) [](ancestors = 180214048)